### PR TITLE
Replace remaining slick-style sql queries with SqlTokens

### PR DESCRIPTION
**Pull request by fm3**
- refactoring: no more slick-style sql queries
- the SQLDAO generic implementations of findAll and findOne now use the access queries, so many more DAOs can use them.
- update access query for ai models

### URL of deployed dev instance (used for testing):
- https://sqlcleanup.webknossos.xyz

### Steps to test:
- Test basic webknossos functionality
- test AI model organization sharing
- update description + name + othersMayEdit of annotations
- transfer annotation to other user
- test editing dataset transformations
- test pausing/unpausing a project
- increase task instances
- test invites, test deleting expired
- test changing user team memberships, experiences
- test revoking user auth token

------
- [x] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)

### CODE REVIEW

File: app/controllers/JobController.scala [hunk 1]
```diff
@@ -80,7 +80,7 @@ class JobController @Inject()(jobDAO: JobDAO,
     for {
       _ <- Fox.successful(())
       jobCountsByState <- jobDAO.countByState
-      workers <- workerDAO.findAll
+      workers <- workerDAO.findAll(GlobalAccessContext)
```
Comment (MichaelBuessemeyer) [inline app/controllers/JobController.scala:79]: guessing here: So now all user can see the status of all workers?

File: app/models/dataset/Dataset.scala [hunk 2]
```diff
@@ -1451,15 +1437,16 @@ class DatasetCoordinateTransformationsDAO @Inject()(sqlClient: SqlClient)(implic
...
+                                            layerName: String): Fox[Seq[CoordinateTransformation]] =
     for {
-      rows <- run(
-        DatasetLayerCoordinatetransformations
-          .filter(r => r._Dataset === datasetId.id && r.layername === layerName)
-          .sortBy(r => r.insertionorderindex)
-          .result).map(_.toList)
-      rowsParsed <- Fox.combined(rows.map(parseRow)) ?~> "could not parse transformations row"
-    } yield rowsParsed
+      r <- run(q"""SELECT _dataset, layerName, type, matrix, correspondences, insertionOrderIndex
```
Comment (MichaelBuessemeyer) [inline app/models/dataset/Dataset.scala:1443]: Just a thought: What about using $columns here? Or do we no want to avoid using $columns?

File: app/controllers/JobController.scala (hunk 1 above)
Comment (fm3) [reply app/controllers/JobController.scala:79]: This was already the case on master (there it just used the findAll that doesn’t consider the access context. This refactoring makes it necessary to explicitly pass the GlobalAccessContext, which makes this more explicit in my opinion :)

File: app/models/dataset/Dataset.scala (hunk 2 above)
Comment (fm3) [reply app/models/dataset/Dataset.scala:1443]: DatasetCoordinateTransformationsDAO doesn’t have `$columns` because it is just a `SimpleSQLDAO`. In the future, we might convert more SimpleSQLDAO to the more feature-rich SQLDAO. This becomes possible now, as it doesn’t require an isDeleted column anymore, but I didn’t go for this change in this PR yet to keep the diff relatively small.

File: app/models/dataset/DataStore.scala [hunk 3]
```diff
@@ -159,10 +148,10 @@ class DataStoreDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext
...
     } yield parsed
 
-  def updateUrlByName(name: String, url: String): Fox[Unit] = {
-    val query = for { row <- Datastores if notdel(row) && row.name === name } yield row.url
-    for { _ <- run(query.update(url)) } yield ()
-  }
+  def updateUrlByName(name: String, url: String): Fox[Unit] =
+    for {
+      _ <- run(q"UPDATE $collectionName SET url = $url WHERE name = $name".asUpdate)
+    } yield ()
```
[the remark this answers is not in the document]
Comment (MichaelBuessemeyer) [reply app/models/dataset/DataStore.scala:154]: Yeah the remark sounds correct to me
